### PR TITLE
OCLOMRS-506: Tests are failing to run on the OpenMRS yokobue agent

### PR DESCRIPTION
**Pull request by danuluma**
# JIRA TICKET NAME:
[Tests are failing to run on the OpenMRS yokobue agent](https://issues.openmrs.org/browse/OCLOMRS-506)

# Summary:
Tests are failing to run on the OpenMRS yokobue agent due to having deprecated version of react-scripts. This PR fixes the bug.

### CODE REVIEW

**Comment by ODORA0** [comment]
@danuluma Hope you noticed the Travis build failure you could look here `src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js`

**Comment by danuluma** [comment]
Hi, I have seen and pushed a fix for it. It should work now.

File: src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js [hunk 1]
```diff
@@ -850,7 +850,7 @@ describe('Add answer mappings to concept', () => {
     moxios.uninstall();
   });
 
-  it('should add all chosen answer mappings', async () => {
```
Comment (dkayiwa) [inline src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js]: Is this related to the react scripts version change?

Comment (danuluma) [reply src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js]: Yes, the test fails if it's wrapped in an async..await block

Comment (dkayiwa) [reply src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js]: Has that got anything to do with the react scripts version change?

Comment (danuluma) [reply src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js]: it fails when using react-scripts >=2.0.0
